glue_lab/glue_ydoc.py

```diff
@@ -128,13 +128,45 @@ def set(self, value: str) -> None:
 
         links: Dict[str, Dict] = {}
         for link_name in link_names:
-            links[link_name] = contents.get(link_name, {})
-            if links[link_name]["_type"] != COMPONENT_LINK_TYPE:
-                for i in range(1, 3):
-                    listName = links[link_name][f"cids{i}"]
-                    links[link_name][f"cids{i}"] = contents.get(listName, {}).get(
+            link: Dict = contents.get(link_name, {})
```

Comment:
Could you add a test like this one?  https://github.com/QuantStack/glue-lab/blob/be0e40a503ae96c01710642e65819c2716310d38/glue_lab/tests/test_ydoc.py#L15-L22
for a session having link data on it.